docs/LEARNING.md

```diff
@@ -1,2 +1,9 @@
 # Learning
 
+ARM64 assembly is a simple assembly language. You might be able to start off with [a reference card](https://courses.cs.washington.edu/courses/cse469/19wi/arm64.pdf).
+
+ARM also provides [digital reference](https://developer.arm.com/documentation) on the ARM64 (AArch64/ARMv8) architecture.
+
+I hope you enjoy learning ARM assembly and working through the exercises. This is a gateway to getting to know your phone or ARM64 based development hardware (the more recent Raspberry Pi models are an example).
```

Comment:
```suggestion
I hope you enjoy learning ARM assembly and working through the exercises.
This is a gateway to getting to know your phone or ARM64 based development hardware (the more recent Raspberry Pi models are an example).
```

Perhaps "We"?

Reply:
We, indeed. :ok_hand: I'll commit that manual suggestion.